opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -994,4 +995,172 @@ mod tests {
                 == AnyValue::String("Updated by FirstProcessor".into())
         );
     }
+
+    #[derive(Debug, Clone)]
+    struct SyncLogExporter {
+        logs: Arc<Mutex<Vec<(LogRecord, InstrumentationLibrary)>>>,
+    }
+
+    #[async_trait::async_trait]
+    impl LogExporter for SyncLogExporter {
+        async fn export(&mut self, batch: LogBatch<'_>) -> LogResult<()> {
+            let mut logs = self.logs.lock().unwrap();
+            for (log_record, instrumentation) in batch.iter() {
+                logs.push((log_record.clone(), instrumentation.clone()));
+            }
+            Ok(())
+        }
+
+        fn shutdown(&mut self) {}
+
+        fn set_resource(&mut self, _resource: &Resource) {}
+    }
+
+    #[test]
+    fn test_simple_processor_sync_exporter_without_runtime() {
+        let exporter = SyncLogExporter {
+            logs: Arc::new(Mutex::new(Vec::new())),
+        };
+        let processor = SimpleLogProcessor::new(Box::new(exporter.clone()));
+
+        let mut record: LogRecord = Default::default();
+        let instrumentation: InstrumentationLibrary = Default::default();
+
+        processor.emit(&mut record, &instrumentation);
+        processor.force_flush().unwrap();
```

Comment:
why flush and shutdown? shutdown alone is good enough right?

Reply:
Yes as of now, force_flush() is empty. Just as safeguard, if we change it in future, this will test it out.

Reply:
since this test is specifically for simple, we don't actually need either flush or shutdown, so they can be removed.